topydo.conf

```diff
@@ -30,3 +30,10 @@ ignore_weekends             = 1
 append_parent_projects      = 0
 ; Add parent contexts when adding sub todo items
 append_parent_contexts      = 0
+
+[colorscheme]
+; priority_colors = A:6,B:3,C:4
```

Comment:
Would be better to replace the numbers with color names.
(And add a reference to a color table for all values <256).